src/ast/semantic_analyser.cpp

```diff
@@ -1360,6 +1409,11 @@ void SemanticAnalyser::visit(AssignMapStatement &assignment)
       }
     }
   }
+  else if (type == Type::ctx)
+  {
+    // bpf_map_update_elem only accepts a pointer to a element in the stack
+    error("Storing context in the map is not supported", assignment.loc);
```

Comment:
So it's not something that can ever work with BPF? The current wording makes it sound like it's just an unimplemented feature of bpftrace.
Maybe something like "Can not assign context to a map" would be better